front/components/assistant/conversation/ConversationViewer.tsx

```diff
@@ -384,31 +384,43 @@ const ConversationViewer = React.forwardRef<
           <Spinner variant="color" size="xs" />
         </div>
       )}
-
-      {conversation &&
-        groupedMessages.map((group) => {
-          return group.map((message) => {
-            return (
-              <MessageItem
-                key={`message-${message.sId}`}
-                conversationId={conversation.sId}
-                hideReactions={hideReactions}
-                isInModal={isInModal}
-                message={message}
-                owner={owner}
-                reactions={reactions}
-                ref={
-                  message.sId === prevFirstMessageId
-                    ? prevFirstMessageRef
-                    : undefined
-                }
-                user={user}
-                isLastMessage={latestPage?.messages.at(-1)?.sId === message.sId}
-                latestMentions={latestMentions}
-              />
-            );
-          });
-        })}
+      {typedGroupedMessages.map((typedGroup, index) => {
```

Comment:
is it expected to not condition on `conversation` anymore?

Reply:
Overlooked this my bad